keps/sig-auth/20190607-certificates-api.md

```diff
@@ -325,8 +377,13 @@ type CertificateSigningRequestStatus struct {
 }
 
 type CertificateSigningRequestCondition struct {
```

Comment:
I'd like to see a LastTransitionTime in this type.

Reply:
Presumably this would be immutable once set for the `Approved`, `Denied` and `Failed` conditions.

Reply:
> Presumably this would be immutable once set for the Approved, Denied and Failed conditions.

I'm not sure I'd add that level of server-side validation

Reply:
I can add LastTransitionTime and backfill on change server-side.